api/tests/__init__.py

```diff
@@ -150,6 +150,13 @@ def setUpClass(cls):
             name="IMN50987345", asset_make=cls.asset_make
         )
 
+        cls.history1 = apps.get_model("core", "History").objects.create(
```

Comment:
It's more appropriate if this object is created in the `HistoryAPITest` test case since it's not shared by other test cases.

Reply:
yap. I get it . Thanks